bc_obps/reporting/tests/models/test_report_product_emission_allocation.py

```diff
@@ -69,6 +69,21 @@ def test_allow_null_description_if_methodology_is_not_other(self):
                 allocation_other_methodology_description=None,
             )
 
+        with pytest.raises(
+            ValidationError,
+            match="Value 'other' is not a valid choice.",
+        ):
+            make(
+                ReportProductEmissionAllocation,
+                report_version=facility_report.report_version,
+                facility_report=facility_report,
+                report_product=report_product,
+                emission_category=emission_category,
+                allocated_quantity=Decimal('300.4151'),
+                allocation_methodology='other',
+                allocation_other_methodology_description='test',
+            )
+
```

Comment:
since 'other' is not in the choices anymore, this will fail on a django model error. It's not really something we need to test - that is covered by the unit tests of the django framework.

we can safely remove this code